docs/source/suite.rst

```diff
@@ -93,9 +128,52 @@ Rajagopal's full body gait model [https://github.com/opensim-org/opensim-models/
 
 Joints and muscle details can be found in Rajagopal's paper [https://ieeexplore.ieee.org/document/7505900]
 
+The model implementation details can be found on our github repo at `myoLeg <https://github.com/MyoHub/myo_sim/tree/dbaecde82a18341661d4eb6d41928f26dc017bdd/leg>`__
+
 .. image:: images/MyoLeg.png
   :height: 200
 
+.. =============   =================================
```

Comment:
@raku-slyu either this need to be done here or on the model README folder. 
1. format needs to be consistent for all the models 
2. location needs to be consistent for all the models too

Reply:
Edit the corresponding table based on the reference paper